modules/installation-vsphere-requirements-user-provisioned.adoc

```diff
@@ -0,0 +1,258 @@
+`StorageProfile.View`
+
+|vSphere vCenter Cluster
+|Always
+|
+[%hardbreaks]
+`Host.Config.Storage`
+`Resource.AssignVMToPool`
+`VApp.AssignResourcePool`
+`VApp.Import`
+`VirtualMachine.Config.AddNewDisk`
+
+|vSphere Datastore
+|Always
+|
+[%hardbreaks]
+`Datastore.AllocateSpace`
+`Datastore.Browse`
+`Datastore.FileManagement`
+
+|vSphere Port Group
+|Always
+|`Network.Assign`
+
+|Virtual Machine Folder
+|Always
+|
+[%hardbreaks]
+`Resource.AssignVMToPool`
+`VApp.Import`
+`VirtualMachine.Config.AddExistingDisk`
+`VirtualMachine.Config.AddNewDisk`
+`VirtualMachine.Config.AddRemoveDevice`
+`VirtualMachine.Config.AdvancedConfig`
+`VirtualMachine.Config.Annotation`
+`VirtualMachine.Config.CPUCount`
+`VirtualMachine.Config.DiskExtend`
+`VirtualMachine.Config.DiskLease`
+`VirtualMachine.Config.EditDevice`
+`VirtualMachine.Config.Memory`
+`VirtualMachine.Config.RemoveDisk`
+`VirtualMachine.Config.Rename`
+`VirtualMachine.Config.Reset[discrete]GuestInfo`
+`VirtualMachine.Config.Resource`
+`VirtualMachine.Config.Settings`
+`VirtualMachine.Config.UpgradeVirtualHardware`
+`VirtualMachine.Interact.GuestControl`
+`VirtualMachine.Interact.PowerOff`
+`VirtualMachine.Interact.PowerOn`
+`VirtualMachine.Interact.Reset`
+`VirtualMachine.Inventory.Create`
+`VirtualMachine.Inventory.CreateFromExisting`
+`VirtualMachine.Inventory.Delete`
+`VirtualMachine.Provisioning.Clone`
+
+|vSphere vCenter Datacenter
+|If the installation program creates the virtual machine folder
+|
+[%hardbreaks]
+`Resource.AssignVMToPool`
+`VApp.Import`
+`VirtualMachine.Config.AddExistingDisk`
+`VirtualMachine.Config.AddNewDisk`
+`VirtualMachine.Config.AddRemoveDevice`
+`VirtualMachine.Config.AdvancedConfig`
+`VirtualMachine.Config.Annotation`
+`VirtualMachine.Config.CPUCount`
+`VirtualMachine.Config.DiskExtend`
+`VirtualMachine.Config.DiskLease`
+`VirtualMachine.Config.EditDevice`
+`VirtualMachine.Config.Memory`
+`VirtualMachine.Config.RemoveDisk`
+`VirtualMachine.Config.Rename`
+`VirtualMachine.Config.ResetGuestInfo`
+`VirtualMachine.Config.Resource`
+`VirtualMachine.Config.Settings`
+`VirtualMachine.Config.UpgradeVirtualHardware`
+`VirtualMachine.Interact.GuestControl`
+`VirtualMachine.Interact.PowerOff`
+`VirtualMachine.Interact.PowerOn`
+`VirtualMachine.Interact.Reset`
+`VirtualMachine.Inventory.Create`
+`VirtualMachine.Inventory.CreateFromExisting`
+`VirtualMachine.Inventory.Delete`
+`VirtualMachine.Provisioning.Clone`
+`VirtualMachine.Provisioning.DeployTemplate`
+`VirtualMachine.Provisioning.MarkAsTemplate`
+`Folder.Create`
+`Folder.Delete`
+|===
+====
+
+Additionally, the user requires some `ReadOnly` permissions, and some of the roles require permission to propagate the permissions to child objects. These settings vary depending on whether or not you install the cluster into an existing folder.
+
+.Required permissions and propagation settings
+[%collapsible]
+====
+[cols="3a,3a,3a,3a",options="header"]
+|===
+|vSphere object
+|Folder type
+|Propagate to children
```

Comment:
Based on the intro text ("some of the roles require permission to propagate the permissions to child objects") I would think that this propagation is something the user has to make happen. But someone who didn't read carefully might take "Propagate to children" = "True" to mean that these automatically propagate. Do they? If so, no problem. But if it's something the user has to make happen I wonder if there's a different way to phrase either the heading or the cell contents to ensure that users do what they need to for it to work.

Reply:
@jeana-redhat thank you for your review and for highlighting this. Based on my review of the [Add a Global Permission](https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.security.doc/GUID-1CE39F3C-0BC5-4CDC-9D62-D8F90644880D.html) VMware document, there is a requirement to select the **Propagate to children** checkbox for the object, if access to objects in the hierarchy are required by the user.

I think that updating the table to state "Required" and "Not required", instead of "True" and "False", is more self-explanatory. Please let me know if this does not resolve the issue. Thanks!

Reply:
Perfect solution :)

Reply:
Great! Thank you for confirming that.